internal/cortex/cortexpb/cortex.proto

```diff
@@ -7,7 +7,7 @@ package cortexpb;
 
 option go_package = "cortexpb";
 
-import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "gogoproto/gogo.proto";
```

Comment:
I wonder if we can do this rewrite in `go.mod` or with some `sed` magic in the Makefile. 

My concern is that the change can be lost the next time we update Cortex.